src/appleseed/foundation/image/color.h

```diff
@@ -66,7 +66,11 @@ class Color
     static const size_t Components = N;
 
     // Constructors.
-    Color();                                    // leave all components uninitialized
+#if _MSC_VER >= 1800
```

Comment:
What if _MSC_VER is undefined? E.g. linux?

Reply:
Also what abour dtors?

Reply:
> What if _MSC_VER is undefined? E.g. linux?

Wow, good point! Will fix.

> Also what abour dtors?

None of these classes have destructors.